src/applications/vaos/tests/e2e/page-objects/AppointmentList/AppointmentListPageObject.js

```diff
@@ -11,25 +30,6 @@ export class AppointmentListPageObject extends PageObject {
       .first()
       .click({ waitForAnimations: true });
 
-    // cy.axeCheckBestPractice();
-
-    return this;
-  }
-
-  validate() {
-    // Wait for appointments to load
-    cy.wait(['@v2:get:appointments']);
-
-    cy.get('h2')
-      .should('be.visible')
-      .and('contain', 'Appointments');
```

Comment:
This query was causes Cypress to crash in CI build. So it was refactored as 'assertAppointmentList'.